README.md

```diff
@@ -28,7 +28,9 @@ Read the list of [contributors](https://github.com/algosup/2024-2025-project-2-s
 
 ## 📚 What is this project ?
 
-<!-- TODO -->
+**Forsaken Planet** is a serious game developed in Godot, focused on the theme of climate change. In the game, players will explore various environments on a mysterious planet, solving puzzles and uncovering clues about the disappearance of an ancient civilization that once thrived there.
```

Comment:
maybe precise the godot version

Reply:
the godot version hasn't been precized in any documents so, IDK what it is.

Reply:
4.3 if my memory is right